File: methodist_app/app/src/main/java/com/example/methodist_mobile_app/data/dto/CreateEventDto.kt
package com.example.methodist_mobile_app.data.dto

import kotlinx.serialization.Serializable

@Serializable
data class CreateEventDto(
    var dateOfEvent: String = "",
    var endDateOfEvent: String = "",
    var formOfEvent: String = "",
    var formOfParticipation: String = "",
    var isApproved: Boolean = false,
    var isChecked: Boolean = false,
    var location: String = "",
    var name: String = "",
    var quantityOfHours: String = "",
    var result: String = "",
    var status: String = "",
    var typeId: String = "",
    var type: String = "",
)

File: Methodist_API/Methodist_API/Dtos/CreateEntity/CreateEventDto.cs
using Methodist_API.Models.DB;

namespace Methodist_API.Dtos.CreateEntity
{
    public class CreateEventDto
    {
        public DateTime DateOfEvent { get; set; }
        public DateTime EndDateOfEvent { get; set; }
        public Guid TypeId { get; set; }
        public bool IsChecked { get; set; } = false;
        public bool IsApproved { get; set; } = false;
        public string Type { get; set; } = string.Empty;
        public string Name { get; set; } = string.Empty;
        public string FormOfParticipation { get; set; } = string.Empty;
        public string FormOfEvent { get; set; } = string.Empty;
        public string Status { get; set; } = string.Empty;
        public string Location { get; set; } = string.Empty;
        public string QuantityOfHours { get; set; } = string.Empty;
        public string Result { get; set; } = string.Empty;
    }
}
